src/drivers/tap_esc/tap_esc.cpp

```diff
@@ -247,6 +247,8 @@ TAP_ESC::init()
 	ConfigInfoBasicRequest   &config = packet.d.reqConfigInfoBasic;
 	memset(&config, 0, sizeof(ConfigInfoBasicRequest));
 	config.maxChannelInUse = _channels_count;
+	/* Enable closed-loop control */
+	config.controlMode = 1;
```

Comment:
I thought that was a there until you had a chance to figure it out. No?  

If it is going to stay let's just be consistent with the naming. The BOARD_ prefix would be a nice to have, but I suspect there are a ton of thing not prefixed. So I will leave it up to you.